numpy/core/src/umath/override.c

```diff
@@ -468,32 +471,34 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
     }
 
     len = PyTuple_GET_SIZE(normal_args);
-    override_args = PyTuple_New(len + 2);
+    override_args = PyTuple_New(len + 3);
     if (override_args == NULL) {
         goto fail;
     }
 
-    Py_INCREF(ufunc);
     /* PyTuple_SET_ITEM steals reference */
-    PyTuple_SET_ITEM(override_args, 0, (PyObject *)ufunc);
+    Py_INCREF(Py_None);
```

Comment:
I was worried about the behaviour of `PyTuple_SetItem` with a NULL in its target position.

I do not need to decref `None`, because unlike `SETITEM`, `SetItem` handles the decref